### PR TITLE
BAEL-3926 - Java Map with case-insensitive keys

### PR DESCRIPTION
**Pull request by omerfinger**

### CODE REVIEW

**Comment by omerfinger** [comment]
There is another pending PR who opened a new module with the exact name

**Comment by eric-martin** [comment]
Retest this please

File: java-collections-maps-3/pom.xml [hunk 1]
```diff
@@ -2,25 +2,39 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <artifactId>java-collections-maps-3</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
-    <name>java-collections-maps-3</name>
-    <packaging>jar</packaging>
```
Comment (eric-martin) [inline java-collections-maps-3/pom.xml:9]: Can you undo your change here? The only change to the pom file should be the new deps and props.

Comment (omerfinger) [reply java-collections-maps-3/pom.xml:9]: Hi, yes I will, at the time this pom wasn't existing, so I created this file. I will delete it.
As for the name element, it said in the authors' documentation it should be deleted.

Comment (omerfinger) [reply java-collections-maps-3/pom.xml:9]: Also, packaging default value is jar.

**Pull request merged**